tests/clims/models/test_project.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+
+import pytest
+from sentry.testutils import TestCase
+from clims.services import ProjectBase, SubstanceBase, FloatField, TextField, IntField
+from clims.models.project import Project
+from django.db import IntegrityError
+
+
+class TestProject(TestCase):
+
+    def setUp(self):
+        self.register_extensible(VampireFangStudyProject)
+        self.register_extensible(VampireFangSample)
+
+    def test_can_create_project(self):
+        project = VampireFangStudyProject(name="project1", organization=self.organization)
+        project.save()
+        Project.objects.get(name=project.name)  # Raises DoesNotExist if it wasn't created
```

Comment:
I'm concerned about using Project class directly in tests. I think the tests should show a reader how to use the classes, and the class Project should not be used by upper layers, it should be wrapped (and hidden) within ProjectBase.